### DISCUSSION TITLE
Icon images on PyInstaller

### DISCUSSION THREAD

**Discussion by Tarufetti** [Akascape/CTkMessagebox#9] [Q&A] [answered]
Hi! I turned my script into an .exe using PyInstaller but it showed an error indicating that there was no 'check.png' on the directory. I solved it adding my own icon, but I wanted to ask where can I find the icons on my computer(I guess I have them already since I downloaded the module using pip).

I'm new to GUIs so I appreciate the help!

**Comment by Akascape** [comment, marked as answer]
@Tarufetti 
You have to pack the whole `Ctkmessagebox` folder (additional folder) in pyinstaller just like you did with customtkinter folder. You can find it under site-packages directory of python lib, or you can download the source code folder separately from the repo.

The main reason is that pyinstaller doesn't pack images or non-python files automatically.
